duckduckgo/Android
# Onboarding Customisation Experiment

Task/Issue URL: https://app.asana.com/0/72649045549333/1203826977705495/f

### Description
Implement Onboarding Customisation experiment where experimental group will see an extra onboarding dialog with DDG feature options.

### Steps to test this PR

_Change all variants to 0 except for the `mj` variant_

_New multiselect dialog_
- Uninstall previous version of the app or clear storage
- Install from this branch
- Welcome animation will appear
- [ ] Check new dialog with DDG feature options appear

_Skip selecting options_
- Follow same instructions as "New multiselect dialog" test
- Tap on `Skip This Step`
- [ ] Check existing welcome cta dialog appears
- [ ] Check pixel `m_onboarding_option_skip` is fired

_Continue without options selected_
- Follow same instructions as "New multiselect dialog" test
- Tap on any option to select it
- Remove the selection tapping again
- Tap on `Continue` button
- [ ] Check existing welcome cta dialog appears
- [ ] Check pixel `m_onboarding_option_skip` is fired

_Continue with options selected_
- Follow same instructions as "New multiselect dialog" test
- Tap on one or more options to select them
- Tap on `Continue` button
- [ ] Check existing welcome cta dialog appears
- [ ] Check pixel `m_onboarding_option_selected` is fired
- [ ] Check pixels for each option selected is also fired. You can find the list of pixels for each option here -> https://app.asana.com/0/0/1204137437551040/f

### UI changes


https://user-images.githubusercontent.com/20798495/228512221-df3caf98-2bda-4f0e-9976-adfc51776f9e.mp4
